app/assets/stylesheets/frontend/views/_history-people.scss

```diff
@@ -16,13 +16,17 @@
       font-weight: bold;
     }
     .term {
-      @include core-14;
-      color: $grey-1;
+      color: $govuk-secondary-text-colour;
+      display: block;
```

Comment:
This is paragraph element - should be display block by default?

Reply:
`.term` is used in a `span` element in the [long list of all of the Foreign Secretaries](https://github.com/alphagov/whitehall/pull/5083/files#diff-c55c1ebf446f9d83ab4a58b430d690b3R183).